doc/metric_spec.md

```diff
@@ -176,6 +177,12 @@ Value is either `1` or `0`.
 The total number of lines for this metric will be the cardinality of `device`.
 
 
+### `ha_cluster_sbd_devices_total`
+
+#### Description
+
+Total count of configured SBD devices.
```

Comment:
`nit`

we should add here also what is the value it returns since most of metric provide it. ( it something imho we are not doing regularly :grin: )

But we should say:

Total count of cofingured SBD devices. Possible values from 0 to N integer or something perhaps.

Reply:
good idea. We should also do this for all the other ones, because actually non of the `*_total` metrics document their value.